components/brave_new_tab_ui/tsconfig.json

```diff
@@ -5,4 +5,4 @@
     "**/*.tsx",
     "**/*.d.ts",
     "../definitions/*.d.ts"
-}
+, "../brave_news/browser/resources/SettingsButton.tsx"}
```

Comment:
this change should be checked again? json seems broken also.

Reply:
Whoops, no! That was added by a move from VSCode. Nice catch!